sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -167,7 +167,7 @@ export abstract class LinkEntity<LinkT extends Receiver | AwaitableSender | Requ
    * A lock that ensures that opening and closing this
    * link properly cooperate.
    */
-  private _openLock: string = generate_uuid();
+  private _openLock: string = `linkEntity-${generate_uuid()}`;
```

Comment:
I was going to use "open" too but the same lock is used for opening and closing.